README.md

```diff
@@ -41,6 +41,19 @@ Example:
             <td>1978</td>
             <td>100</td>
           </tr>
+          <tr>
+            <td>Micheal</td>
+            <td>Jackson</td>
+            <td>1964</td>
```

Comment:
MJ was born 1958. For dates to showcase what table sort can do, I think we can change Date to the format: 1958/8/29 and use different delimiters for the different people e.g 1958-8-29 1958.8.29 etc. . So if we change all the dates in the table to this format and perhaps to real people: like for example changing Bach Frank to Benjamin Franklin or Sebastian Bach then you can put a real date.

Ben 10 is fine though can just make up a date for him; maybe when the show started?, or something.